cmd/openscap-plugin/oscap/oscap.go

```diff
@@ -5,10 +5,36 @@ package oscap
 import (
 	"fmt"
 	"os/exec"
+	"path/filepath"
 
+	"github.com/complytime/complytime/cmd/openscap-plugin/config"
 	"github.com/hashicorp/go-hclog"
 )
 
+func executeCommand(command []string) ([]byte, error) {
+	cmdPath, err := exec.LookPath(command[0])
+	if err != nil {
+		return nil, fmt.Errorf("command not found: %s", command[0])
```

Comment:
Maybe wrapping the original error if it provide additional context?
```suggestion
		return nil, fmt.Errorf("command not found: %s: %w", command[0], err)
```